dependencies/zycore.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-git]
+# This is a fallback for plain clone (no submodules available).
+url = https://github.com/zyantific/zycore-c.git
+# Sync revision to `git submodule status zycore`.
```

Comment:
Can you add a line reminding us to do that in https://github.com/zyantific/zydis/blob/master/assets/version-bump-checklist.txt please? Otherwise we'll probably forget.